.github/workflows/update-contribs.yml

```
@@ -0,0 +1,44 @@
+name: Update Contributors
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+    run-meta:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Check out the code
+          uses: actions/checkout@v3
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: '3.10'
+        - name: Upgrade pip
+          run: |
+            # install pip=>20.1 to use "pip cache dir"
+            python3 -m pip install --upgrade pip
+            echo "$PWD"
+            git branch
+        - name: Install pyosmeta and run update contribs
+          env:
+            GITHUB_TOKEN: ${{ github.token }}
+          run: |
+            pip install pyosmeta@git+https://github.com/pyopensci/update-web-metadata
+            echo "$PWD"
+            python3 scripts/parse-contributors.py
+        - name: Commit Files
+          run: |
+            git config --global user.email "leah@pyopensci.org"
+            git config --global user.name "Leah"
+            git status
+            git add _data/contributors.yml
+            git commit -m "Update: Contributor file update"
+        - name: Create Pull Request
+          uses: peter-evans/create-pull-request@v5
+          with:
+            base: main
+            branch: contribs
```

```suggestion
        - name: Create Pull Request
          uses: peter-evans/create-pull-request@v5
          with:
            add-paths: _data/contributors.yml
            author: Leah <leah@pyopensci.org>
            base: main
            branch: contribs
            commit-message: "Update: Contributor file update"
```

Seems to me its simpler and cleaner to just do this with the action's built-in features instead of manual git commands, no?